numpy/lib/npyio.py

```diff
@@ -2235,13 +2237,18 @@ def ndfromtxt(fname, **kwargs):
 
     """
     kwargs['usemask'] = False
+    warnings.warn(
+        "np.ndfromtxt is deprecated",
+        DeprecationWarning, stacklevel=2)
```

Comment:
Needs a comment above the warning with the date, version, gh-issue format

Reply:
Should also recommend a replacement, perhaps `` `ndfromtxt` is a deprecated alias of `genfromtxt`, prefer the latter ``